config/default/kustomization.yaml

```diff
@@ -1,13 +1,6 @@
 # Adds namespace to all resources.
 namespace: csi-addons-system
 
-# Value of this field is prepended to the
-# names of all resources, e.g. a deployment named
-# "wordpress" becomes "alices-wordpress".
-# Note that it should also match with the prefix (text before '-') of the namespace
-# field above.
-namePrefix: csi-addons-
```

Comment:
This looks to be in the right place, needs to be removed from the manager and rbac folder but can be revisited later as it will also do the same thing, we need to change few things on how we are doing the Kustomization in makefile